builtin/game/knockback.lua

```diff
@@ -23,6 +23,9 @@ local function vector_absmax(v)
 end
 
 core.register_on_punchplayer(function(player, hitter, time_from_last_punch, tool_capabilities, unused_dir, damage)
+	if not hitter then
+		return
```

Comment:
Why early-return here? Couldn't you just use `dir` (currently unused) when no hitter is available? Mods that don't want knockback could simply pass the null vector.

Reply:
Good point. I missed this.